.github/workflows/test.yml

```diff
@@ -329,6 +329,15 @@ jobs:
         with:
           timezoneLinux: ${{ needs.setup.outputs.timezone }}
 
+      - name: Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ vars.NODE_VERSION }}
+
+      - name: Create node_modules dir
```

Comment:
This is not required as `npm run ci:all` will create.
If there is a delay it might be temporary in nature pertaining to GH public runners.